src/traversals/maxadjvisit.jl

```diff
@@ -161,6 +161,65 @@ function mincut(
 end
 
 
+function mincut_new(
+    g::AbstractGraph,
+    distmx::AbstractMatrix{T}=weights(g)
+) where T
+
+    U = eltype(g)
+    colormap = zeros(Int8, nv(g))   ## 0 if unseen, 1 if processing and 2 if seen and closed
+    parities = falses(nv(g))
+    bestweight = typemax(T)
+    cutweight = zero(T)
+    visited = zero(Int)             ## number of vertices visited
+    # vertices = Vector{U}()        ## not sure if this extra info is necessary to be stored
```

Comment:
@sbromberger do you need this information?

Reply:
I don't think so. Let's leave it out.